packages/compiler/src/myst.ts

```diff
@@ -36,5 +39,48 @@ export function parsePanel(contents: string): Array<MystCard> {
     parsedCard.body = body;
     splitCards.push(parsedCard);
   }
-  return splitCards;
+
+  let myPanel: MystPanel = {cards: splitCards}
+  if (Object.keys(panelStyling).length !== 0) {
+    myPanel = {...myPanel, style: panelStyling}
+  }
+  return myPanel
+}
+
+export function parsePanelStyling(contents: string) {
+  const contentLines = contents.split("\n")
+  // TO ASK: This regex was a PITA. 
+  // Is there a way to get it to do this without having to split contents by newline?
+  const panelStylingRegex = /:[a-z]+:\s[a-z\s\-0-9]+/i;
+  let panelContentLine;
+  let styles: MystPanelStyling = {} as MystPanelStyling;
+  // panel styling is only at the beginning of the panels
+  for (let i = 0; i < contentLines.length; i++) {
+    // stop processing lines once individual cards declared
+    if (!contentLines[i].startsWith(":")) {
+      panelContentLine = i
+      break
+    } else {
+      const valueMatch = contentLines[i].match(panelStylingRegex)
+      if (valueMatch !== null && valueMatch.length === 1) {
+        const splitValues: Array<string> = valueMatch[0].split(": ")
+        if (splitValues.length === 2) {
+          const elementType: string = splitValues[0].substring(1) // remove first :
+          const addtClasses: string = splitValues[1]
+          // NOTE: This prop assignment does *not* enforce typing...
+          // How can you dynamically set a prop only if it's a valid prop according to type?
+          styles = {...styles, [elementType]: addtClasses}
```

Comment:
I suspect it might be easier to just parse out the key/value pairs and then extract out the styling information we want to support after that's done. The code to do the former could probably be reused across many parts of the myst support (as is the case in the python myst parser linked above).